configs/components/fesom/fesom-2.6.yaml

```
@@ -43,6 +43,9 @@ fesom:
       "2.6.8":
         branch: "2.6.8"
 
+      "2.6.9":
+        branch: "2.6.8"
```

This is incorrect right?

```suggestion
      "2.6.9":
        branch: "2.6.9"
```

---

oh yes!